### ISSUE TITLE
Equation numbering

### ISSUE THREAD

**Issue by davidanthoff** [JuliaDocs/Documenter.jl#975]
It would be great if we had some syntax to do equation numbering.

**Comment by mortenpi** [comment]
It turns out we already support this :slightly_smiling_face:. [MathJax can do equation numbering](http://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering) and Documenter has the option enabled. You can get equation numbers and references (won't work across pages though) by doing something like this:

````markdown
```math
\begin{equation}
  \label{foo}
  E = mc^2
\end{equation}
```

Equation \eqref{foo} means that ...
````

We should probably document this in the LaTeX syntax section.

**Comment by simonbyrne** [comment]
Apparently this is not supported by KaTeX: https://github.com/KaTeX/KaTeX/issues/350

**Comment by lmiq** [comment]
> It turns out we already support this 🙂. [MathJax can do equation numbering](http://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering) and Documenter has the option enabled. You can get equation numbers and references (won't work across pages though) by doing something like this:
> 
> ```math
> \begin{equation}
>   \label{foo}
>   E = mc^2
> \end{equation}
> ```
> 
> Equation \eqref{foo} means that ...
> We should probably document this in the LaTeX syntax section.


Sorry reviving this thread. Does this work? It seems not to be working.

**Comment by goerz** [comment]
It works… See, for example, https://juliaquantumcontrol.github.io/GRAPE.jl/dev/background/

You can have a look at [what the setup is](https://github.com/JuliaQuantumControl/GRAPE.jl/blob/2f4ae7dd2c709256f1613cc5273c018327c15ff2/docs/make.jl#L94-L103).
